### ISSUE TITLE
Vertical vim split divider doesn't match screenshots

### ISSUE THREAD

**Issue by kaicataldo** [trevordmiller/nova-vim#75]
Hi! First of all, thanks for your work on this :) 

I was just trying out nova-vim and noticed that vertical split dividers don't match the horizontal dividers or the screenshots on the website. Do you know what I might be doing wrong?

Here's what I'm seeing:
<img width="912" alt="screen shot 2017-08-12 at 12 15 05 am" src="https://user-images.githubusercontent.com/7041728/29237798-e690da34-7ef3-11e7-9660-0351f064867b.png">

I just downloaded the iTerm2 theme and am using and also just ran `:PlugUpdate`, so both my iTerm2 colors and the Vim plugin should be up to date. Thanks!

**Comment by trevordmiller** [comment]
Hi @kaicataldo, thanks for trying out Nova :) Since https://github.com/trevordmiller/nova-vim/pull/69 which was released in `3.9.0`, both the horizontal dividers should be `backgroundShade` from `nova-colors`, which looks like this:

![image](https://user-images.githubusercontent.com/5497885/29243746-0710682c-7f64-11e7-9e97-e40e5037feb7.png)

So, your vertical divider color is correct, but since you are using vim airline, it overrides the native status color (the horizontal dividers); I'll update this issue title to reflect that the vim airline code needs to be updated to use `backgroundShade` to match the built-in status bar like the screenshot above ^. I don't use airline - but @alexlafroscia added support for it to this plugin; Alex, would you mind updating the plugin so that the airline status better matches the latest native status from https://github.com/trevordmiller/nova-vim/pull/69?

The screenshots on the website were taken before this change so I need to update them, thanks for the reminder! I've made an issue for this: https://github.com/trevordmiller/nova-vim/issues/76.

**Comment by alexlafroscia** [comment]
Yeah, I can try. If I make most of the status like that dark, what color should I make the "arrows"? There are two different colors for them in normal mode, plus the alternate colors for the other modes.

Current config:
<img width="752" alt="screen shot 2017-08-12 at 10 07 20 pm" src="https://user-images.githubusercontent.com/1645881/29245525-ad37c6dc-7faa-11e7-96fd-e0dd580d166f.png">

**Comment by trevordmiller** [comment]
@alexlafroscia I'll defer to your opinion since you use airline; if you want to keep it as is that's cool too, although the reason I switched to the darker status was for contrast as I had recieved a lot of feedback about it. Specifically the right pane's case of this screenshot in the airline version:
![image](https://user-images.githubusercontent.com/5497885/29246111-4a0bb7b0-7fac-11e7-93d2-ce3955669693.png)

**Comment by kaicataldo** [comment]
@trevordmiller Gotcha - thanks for the quick and thorough response and for you work!

**Comment by trevordmiller** [comment]
@alexlafroscia No worries. Let me know if you don't have time / don't want to and I'll take care of it; just wanted to give you a chance to continue to contribute since you did such a great job with the original airline setup and have been a great supporter of Nova! Thank you :)

**Comment by trevordmiller** [comment]
@alexlafroscia If needed, we could define some more grays between `backgroundShade` and `background` in `nova-colors` as well if needed: https://github.com/trevordmiller/nova-colors/blob/master/src/index.js#L28-L29. But my preference would be if the default airline could use the `backgroundShade` for the background color with `gray3` for foreground color by default and `userCurrentState` for foreground color when active  as #69 implemented for standard status bars.

**Comment by dbjsy** [comment]
Hi Guys,

I couldn't conclude from the above - does it mean the airline bars meant to look like the following:

![image](https://user-images.githubusercontent.com/8529293/29667803-b1e6859a-88d5-11e7-803d-50749bad790d.png)

It is within tmux, vim 8.

Thanks in advance for answers.

**Comment by trevordmiller** [comment]
@dbjsy Thanks! No, your setup looks wonky, like it is mixing the native status line and airline. Maybe when @alexlafroscia is done making his update to have the airline match the native status line it will fix this, but I don't use airline so I'm not sure. This is how a non-airline setup looks - which in my mind is the "correct" look of nova:

![image](https://user-images.githubusercontent.com/5497885/29675285-f8f750c8-88b1-11e7-82d9-981a19efb244.png)

I'd like to see airline updated to match this as much as possible.

**Comment by trevordmiller** [comment]
> " Any chances for this theme to be implemented for oh-my-zsh" - @dbjsy 

Doesn't zsh just inherit from your terminal theme like other shells? In that case we have plugins for Hyper and iTerm2 on https://trevordmiller.com/projects/nova; [Gnome terminal in the works](https://github.com/agarrharr/gnome-terminal-nova) by @agarrharr and will be added to the website once complete. If zsh requires it's own theme, I personally wont be building it but if someone else makes a good version using `nova-colors` correctly as you can see from the [other nova plugins](https://github.com/trevordmiller?utf8=%E2%9C%93&tab=repositories&q=nova-&type=&language=), I'll gladly add it to the website!

**Comment by dbjsy** [comment]
@trevordmiller thank you for your response! Ok I will wait for the update then :)

Meanwhile, as I am a ruby programmer, I noticed there is no coloring for string values here. Is that something that deserves a new issue being created?

![image](https://user-images.githubusercontent.com/8529293/29676159-3b01882a-88ef-11e7-8ea0-c5af73303a0d.png)

You can see here that string values are not recognized and so not colored (comparing to your screenshot above)

**Comment by agarrharr** [comment]
@trevordmiller I use zsh, and I believe that it just inherits the colors from the terminal emulator.

**Comment by alexlafroscia** [comment]
I use ZSH and oh-my-zsh, I don't have these black bits interspersed in the Airline statusbar

**Comment by trevordmiller** [comment]
"as I am a ruby programmer, I noticed there is no coloring for string values here. Is that something that deserves a new issue being created?" - @dbjsy 

This plugin implements all Vim native highlight groups. It looks like Ruby strings consider the inner part the string (hence the cyan) and the quotes as "special" so they inherit the orange. See:
https://github.com/trevordmiller/nova-vim/blob/master/src/index.js#L21-L269

So I don't consider this an issue as that is how Ruby inherits from the Vim highlight groups.

**Comment by dbjsy** [comment]
@trevordmiller it is probably my lack of knowledge of insights of vim ;), thus thank you for your explanation.

**Comment by alexlafroscia** [comment]
Hey, I don't think I'm going to have time for this. It's pretty easy to adjust the colors in the airline theme, it's just time consuming (and personally I like the way it looks right now). But if you want to change it @kaicataldo it should be pretty easy.
